### ISSUE TITLE
Masks

### ISSUE THREAD

**Issue by AlexTS1980** [UCSD-AI4H/COVID-CT#35]
Can you provide masks (ground glass opacity, consolidation) to evaluate a segmentation algorithm?

**Comment by shambhavimalik** [comment]
> Can you provide masks (ground glass opacity, consolidation) to evaluate a segmentation algorithm?

Yes, can you please provide the corresponding masks? I want to use the segmentation workflow (res UNet) provided in your research paper.
